plugins/org.eclipse.fordiac.ide.library.model/src/org/eclipse/fordiac/ide/library/model/util/ManifestHelper.java

```diff
@@ -466,19 +466,28 @@ public static boolean saveManifest(final Manifest manifest) {
 	}
 
 	/**
-	 * Sorts dependencies and saves the {@link Manifest}
+	 * Sorts dependencies of the {@link Manifest} <br>
+	 * Note: Does not save the manifest.
 	 *
 	 * @param manifest specified manifest
-	 * @return {@code true} if it was saved successfully, else {@code false}
+	 * @return {@code true} if sorting was necessary, else {@code false}
 	 */
-	public static boolean sortAndSaveManifest(final Manifest manifest) {
-		if (manifest.getDependencies() != null) {
-			// ensure dependencies are sorted (can't use EList.sort())
-			final var dependencies = new LinkedList<>(manifest.getDependencies().getRequired());
-			manifest.getDependencies().getRequired().clear();
-			dependencies.forEach(d -> ManifestHelper.addDependency(manifest, d));
+	public static boolean sortManifestDependencies(final Manifest manifest) {
+		if (manifest.getDependencies() == null) {
+			return false; // nothing to do
+		}
+		// ensure dependencies are sorted (can't use EList.sort())
+		final var dependencies = new ArrayList<>(manifest.getDependencies().getRequired());
+		manifest.getDependencies().getRequired().clear();
+		dependencies.forEach(d -> ManifestHelper.addDependency(manifest, d));
+		int index = 0;
+		for (final Required req : manifest.getDependencies().getRequired()) {
+			if (req != dependencies.get(index)) {
+				return true;
+			}
+			index++;
 		}
```

Comment:
Could you not simply use:
```suggestion
		return !dependencies.equals(manifest.getDependencies().getRequired());
```